src/Components/ReflectionLibrary/ReflectionLibrary.test.js

```diff
@@ -37,27 +36,26 @@ describe('ReflectionLibrary', ()=>{
     });
   });
 
+  // NOTE: Integration test below needs additional assitance
```

Comment:
This is a good point of reference for specific feedback on the project when we turn it in. Hopefully Khalid or Leta can shed some light on what is happening here.